.github/workflows/cross-chain-arbitrum.yml

```
@@ -0,0 +1,238 @@
+name: Cross chain Arbitrum
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    branches:
+      - main
+    paths:
+      - "cross-chain/arbitrum/**"
+  workflow_dispatch:
```

I think we are missing `pull_request` trigger here.
```suggestion
  pull_request:
  workflow_dispatch:
```